spec/paperclip/storage/fog_spec.rb

```diff
@@ -475,8 +506,52 @@ def custom_method
           assert_equal @dummy.avatar.fog_credentials, @dynamic_fog_credentials
         end
       end
-    end
 
+      context "with AES256 encryption" do
+        before do
+          rebuild_model(@options.merge(fog_encryption: :aes256))
+          @dummy = Dummy.new
+          @dummy.avatar = @file
+        end
+
+        it 'sets the @fog_encryption instance variable' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.